snake-kmp-benchmarks/src/jvmMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/benchmark/SnakeyamlEngineJvmLoadingTimeBenchmark.kt

```diff
@@ -0,0 +1,52 @@
+package it.krzeminski.snakeyaml.engine.kmp.benchmark
+
+import kotlinx.benchmark.*
+import okio.Path.Companion.toPath
+import okio.buffer
+import okio.use
+import org.snakeyaml.engine.v2.api.LoadSettings
+import org.snakeyaml.engine.v2.api.YamlUnicodeReader
+import org.snakeyaml.engine.v2.composer.Composer
+import org.snakeyaml.engine.v2.constructor.BaseConstructor
+import org.snakeyaml.engine.v2.constructor.StandardConstructor
+import org.snakeyaml.engine.v2.parser.ParserImpl
+import org.snakeyaml.engine.v2.scanner.StreamReader
+
+@State(Scope.Benchmark)
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(BenchmarkTimeUnit.MILLISECONDS)
+class SnakeyamlEngineJvmLoadingTimeBenchmark {
```

Comment:
It should be displayed as a separate series on the JVM graph in https://krzema12.github.io/snakeyaml-engine-kmp-benchmarks/dev/bench/

Reply:
Yes, it will be. Each benchmark is displayed as a separate chart by default (you can modify the index.html in gh-pages branch if you need otherwise)

Reply:
> you can modify the index.html in gh-pages branch if you need otherwise

I tried tinkering with JavaScript to have both JVM series on a single graph, but I lost the battle 😅 Maybe another time! This is the feature request on the GitHub action's side: https://github.com/benchmark-action/github-action-benchmark/issues/175